go.sum

```diff
@@ -185,10 +185,6 @@ github.com/hydrogen18/memlistener v0.0.0-20141126152155-54553eb933fb/go.mod h1:q
 github.com/imkira/go-interpol v1.1.0/go.mod h1:z0h2/2T3XF8kyEPpRgJ3kmNv+C43p+I/CoI+jC3w2iA=
 github.com/inconshreveable/mousetrap v1.0.0/go.mod h1:PxqpIevigyE2G7u3NXJIT2ANytuPF1OarO4DADm73n8=
 github.com/influxdata/influxdb1-client v0.0.0-20191209144304-8bf82d3c094d/go.mod h1:qj24IKcXYK6Iy9ceXlo3Tc+vtHo9lIhSX5JddghvEPo=
-github.com/irfansharif/allocator v0.0.0-20210807031756-96fda6c95716 h1:T/EaBUtccHVFTpnwX63qzIaBABfnOhZbCFYseyRIzhM=
-github.com/irfansharif/allocator v0.0.0-20210807031756-96fda6c95716/go.mod h1:V+ekc19OeFYTKni/hHRJniHTNg6sACfmdKP3npD/vvc=
-github.com/irfansharif/or-tools v0.0.0-20210806150710-7bf371607f5d h1:m/iQkO+XU8XYoozqrHTdDufMkCzRJzuQWZIt0QVZWAM=
-github.com/irfansharif/or-tools v0.0.0-20210806150710-7bf371607f5d/go.mod h1:R/dumbSyTGDtltABqk8bHM3YHisdUCW46mEc1eSqCkY=
 github.com/irfansharif/solver v0.0.0-20211002024421-11071354d9bd h1:KFwYhOTlzh8jXabSVs0MWZhoarPOPnEG+YutvnCX0QI=
```

Comment:
Don't we need these?

Reply:
Bro its something that it automatically picked up

Reply:
Will remove this file